wgsl/index.bs

```diff
@@ -18494,6 +18709,559 @@ All synchronization functions [=shader-creation error|must=] only be invoked in
     space.
 </table>
 
+## Subgroup Built-in Functions ## {#subgroup-builtin-functions}
+
+See [[#subgroup-ops]].
+
+Calls to these functions:
+* [=shader-creation error|Must=] only be used in a [=fragment=] or [=compute=] shader stage.
+* [=Trigger=] a [=trigger/subgroup_uniformity=] [=diagnostic=] if [=uniformity analysis=]
+    cannot prove the call is in [=uniform control flow=].
+
+### `subgroupAdd` ### {#subgroupadd-builtin}
+
+<table class='data builtin'>
+  <tr algorithm="subgroupAdd">
+    <td style="width:10%">Overload
+    <td class="nowrap">
+      <xmp highlight=wgsl>
+        @must_use fn subgroupAdd(e : T) -> T
+      </xmp>
+  <tr>
+    <td>Preconditions
+    <td>`T` is [=type/concrete=] [=numeric scalar=] or [=numeric vector=]
+  <tr>
+    <td>Description
+    <td>Reduction operation.<br>
+
+    Returns the sum of `e` among all [=subgroups/active=] invocations in the [=subgroup=].
+</table>
+
+#### `subgroupExclusiveAdd` #### {#subgroupexclusiveadd-builtin}
```

Comment:
I would put the inclusive/exclusive forms out at the same section/header level as the main function

Reply:
This way is fine.